integration-libs/opf/checkout/components/opf-checkout-payment-and-review/opf-checkout-payment-and-review.component.html

```diff
@@ -0,0 +1,126 @@
+<!-- For B2B Account payment, OPF should not be displayed -->
+<ng-container *ngIf="(paymentType$ | async)?.code !== 'ACCOUNT'">
```

Comment:
[QUESTION]

Just curious: What when it's B2B site? Do we display an empty checkout page? (because ngIf is false?)